internal/participate/contribute-to-wiki/getting_started/getting_started.md

```diff
@@ -0,0 +1,32 @@
+---
+keywords:
+- getting started
+description: How to start editing IOTA Wiki pages.
+image: /img/participate.svg
+---
+
+# Getting Started
+
+If you want to edit a document to elaborate on a topic or fix a typo, you need to edit its source file. General topics and most of the user-oriented content is stored directly in the [IOTA Wiki's repository] (https://github.com/iota-community/iota-wiki), but most of its content comes from repositories of projects within IOTA ecosystem:
```

Comment:
I think we can refactor this whole page. No one needs to find the source to edit a typo. Just use edit this page on the bottom left of every wiki page. Or what did you try to achieve here?

Reply:
I thought It was broken. Reverted the change.

Reply:
Only the editor was/is broken 😓